crates/stackable-operator/src/commons/telemetry.rs

```diff
@@ -0,0 +1,87 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use url::Url;
+
+use crate::time::Duration;
+
+// NOTE (@Techassi): We might want to rename this to OpenTelemetryConfig to
+// avoid confusion about business telemetry.
+#[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TelemetryConfig {
```

Comment:
You are right. I actually started adding those but didn't push them yet. Will do so asap.

Reply:
Good idea. A suggestion for renaming:

```diff
- telemetry::TelemetryConfig
+ open_telemetry::Config
```

That of course make sense if the struct is qualified with the `open_telemetry` module in the name. If we want to import the struct, then it might conflict.

Reply:
Fine with whatever you decide, i like prefixing, would just like to have it decided before we roll it out :)

Reply:
Renamed to `OpenTelemetryConfig` in 8eb99eb